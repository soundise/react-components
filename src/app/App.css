@import url('https://fonts.googleapis.com/css2?family=Inter:wght@700&display=swap');

body {
    margin: 0;
    font-family: 'Inter', 'system-ui', 'sans-serif';
    text-align: center;
    color: #FFFFFF;
    background: #000000;
}

.wrapper {
    max-width: 48rem;
    min-height: calc(100vh - 4rem);
    padding: 2rem;
    margin: 0 auto;
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
}

.image {
    width: 100%;
    display: block;
}

.heading {
    width: 100%;
    padding-bottom: 1rem;
    border-bottom: 1px solid #26282E;
    margin-top: 1.5rem;
    margin-bottom: 1rem;
}

.description {
    margin: 0;
}
