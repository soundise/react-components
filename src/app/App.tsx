import React from 'react';
import styled from 'styled-components';

export const App = () => (
  <Wrapper className="wrapper">
    <a href="https://soundise.com" target="_blank" rel="noreferrer">
      <Image
        src="https://assets.soundise.com/headers/header-github-readme-dark.svg"
        alt="Soundise React component library"
      />
    </a>
    <Heading className="heading">react-components</Heading>
    <Description>Soundise React component library</Description>
  </Wrapper>
);

const Wrapper = styled.div`
  max-width: 48rem;
  min-height: calc(100vh - 4rem);
  padding: 2rem;
  margin: 0 auto;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
`;

const Image = styled.img`
  width: 100%;
  display: block;
`;

const Heading = styled.h1`
  width: 100%;
  padding-bottom: 1rem;
  border-bottom: 1px solid #26282e;
  margin-top: 1.5rem;
  margin-bottom: 1rem;
  text-align: center;
  font-weight: 700;
`;

const Description = styled.p`
  font-weight: 700;
`;
