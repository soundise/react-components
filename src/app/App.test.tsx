import React from 'react';
import { render, cleanup } from 'tests';
import { App } from 'app';

afterEach(cleanup);

describe('App', () => {
  it('should render the App component', () => {
    const { container } = render(<App />);

    expect(container).toBeTruthy();
    expect(container).toMatchSnapshot();
  });
});
