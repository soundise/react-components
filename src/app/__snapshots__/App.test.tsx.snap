// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render the App component 1`] = `
<div>
  <div
    class="sc-dkrFOg fqAvKa wrapper"
  >
    <a
      href="https://soundise.com"
      rel="noreferrer"
      target="_blank"
    >
      <img
        alt="Soundise React component library"
        class="sc-hLBbgP fBkuHm"
        src="https://assets.soundise.com/headers/header-github-readme-dark.svg"
      />
    </a>
    <h1
      class="sc-eDvSVe bDCePJ heading"
    >
      react-components
    </h1>
    <p
      class="sc-jSUZER cPeVAA"
    >
      Soundise React component library
    </p>
  </div>
</div>
`;
